src/containers/LabelingCanvas/useLabels.test.js

```diff
@@ -38,9 +37,8 @@ jest.mock('@/actions/markup', () => mockMarkupActions)
 jest.mock('@/actions/ui', () => mockUiActions)
 
 const mockLabel = new Label(1, 2, 3, 4)
-const mockArea = new Area(1, 2, 3, 4)
 const mockLabels = [mockLabel]
-const mockObjects = [mockLabel, mockArea]
+const mockObjects = [mockLabel]
```

Comment:
looks like we can operate without `mockObjects` now, right?

Reply:
Fixed